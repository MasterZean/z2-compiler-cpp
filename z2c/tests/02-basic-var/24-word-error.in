val a = 65535u16
val b = 65536u16