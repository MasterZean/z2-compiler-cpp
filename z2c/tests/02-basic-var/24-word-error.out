uint16 a = 65535u;
DummyClass(2, 9): error:
	'Word' literal constant too big
